EdvClient.js

```diff
@@ -247,7 +247,7 @@ export class EdvClient {
         doc.meta = {};
       }
       // specify stream information
-      doc.meta.stream = {
+      doc.stream = {
```

Comment:
There's a line above this one I can't leave a comment on (246) where we are adding `doc.meta` if it doesn't exist. I suspect we don't need to do that anymore.

Reply:
Removed.